src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -157,21 +169,8 @@
 
     this.flushService = new FlushService(this, this.channelCache, this.isTestMode);
 
-    if (ENABLE_PERF_MEASUREMENT) {
-      metrics = new MetricRegistry();
-      blobSizeHistogram = metrics.histogram(MetricRegistry.name(name, "blob", "size", "histogram"));
-      blobRowCountHistogram =
-          metrics.histogram(MetricRegistry.name(name, "blob", "row", "count", "histogram"));
-      cpuHistogram = metrics.histogram(MetricRegistry.name(name, "cpu", "usage", "histogram"));
-      flushLatency = metrics.timer(MetricRegistry.name(name, "flush", "latency"));
-      buildLatency = metrics.timer(MetricRegistry.name(name, "build", "latency"));
-      uploadLatency = metrics.timer(MetricRegistry.name(name, "upload", "latency"));
-      registerLatency = metrics.timer(MetricRegistry.name(name, "register", "latency"));
-      uploadThroughput = metrics.meter(MetricRegistry.name(name, "upload", "throughput"));
-      inputThroughput = metrics.meter(MetricRegistry.name(name, "input", "throughput"));
-      metrics.register(MetricRegistry.name("jvm", "memory"), new MemoryUsageGaugeSet());
-      metrics.register(MetricRegistry.name("jvm", "threads"), new ThreadStatesGaugeSet());
-      SharedMetricRegistries.add("Metrics", metrics);
+    if (this.parameterProvider.hasEnabledSnowpipeStreamingJmxMetrics()) {
```

Comment:
It contains both jmx and jvm metrics
```suggestion
    if (this.parameterProvider.hasEnabledSnowpipeStreamingMetrics()) {
```